components/sensor/tem3200.rst

```diff
@@ -0,0 +1,116 @@
+TE-M3200 Pressure Sensor
+===========================================
+
+.. seo::
+    :description: Instructions for setting up TE-M3200 pressure sensors with ESPHome
+    :image: tem3200.jpg
+    :keywords: TEM3200 TE-M3200
+
+The ``tem3200`` sensor platform allows you to use your TE-M3200 (`datasheet <https://www.te.com/commerce/DocumentDelivery/DDEController?Action=showdoc&DocId=Data+Sheet%7FM3200%7FA20%7Fpdf%7FEnglish%7FENG_DS_M3200_A20.pdf>`__,
+`product page <https://www.te.com/en/product-CAT-PTT0068.html>`__) pressure sensors with ESPHome.
+
+.. figure:: images/tem3200.jpg
+    :align: center
+    :width: 50.0%
+
+    TE-M3200 Pressure Sensor.
+
+
+Configuration
+-------------
+:ref:`I²C <i2c>` bus is required to be set up in your configuration for this sensor to work.
+
+.. code-block:: yaml
+
+    # Example configuration entry
+    i2c:
+
+    sensor:
+      - platform: tem3200
+        raw_pressure:
+          name: "Raw Pressure"
+          id: raw_pressure
+
+        temperature:
+          name: Temperature
+
+      - platform: copy
+        source_id: raw_pressure
+        name: Pressure
+        state_class: measurement
+        device_class: pressure
+        unit_of_measurement: psi
+
+        filters:
+          - calibrate_linear:
+            - 1000 -> 0.0
+            - 15000 -> 100.0
```

Comment:
Dont need extra info in the example. If people read the page properly they will see the linear calibration suggested below. If you want, you can include the `copy` sensor in the calibration section

```suggestion
    # Example configuration entry
    sensor:
      - platform: tem3200
        raw_pressure:
          name: "Raw Pressure"
        temperature:
          name: Temperature
```